array-manipulation-1/src/tail.js

```diff
@@ -0,0 +1,18 @@
+/* exported tail */
+function tail(array) {
+  var tailEl = [];
+  for (var i = 0; i < array.length; i++) {
+    if (i !== 0) {
```

Comment:
Instead of having this conditional in the loop, you can adjust your initialization to start at `1` rather than. `0`, and remove the need for this.